src/main/java/com/hwr/hwrzeiterfassung/database/controller/ProjectController.java

```diff
@@ -0,0 +1,18 @@
+package com.hwr.hwrzeiterfassung.database.controller;
+
+import com.hwr.hwrzeiterfassung.database.repositorys.ProjectRepository;
+import com.hwr.hwrzeiterfassung.database.tables.Project;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/project")
+public class ProjectController {
+    @Autowired
+    private ProjectRepository projectRepository;
+
+    public Project getProjectById(int projectId){
+        return projectRepository.findById(projectId).get();
```

Comment:
is das nicht unnötig, da es eigentlich nur eine andere genauso lange Methode aufruft

Reply:
Neue Idee: Ich könnte da ein Troubleshooting wie in PR #51 einfügen und dafür meinen doch eher unpäzisen Try-Catch-Block im `TimeEntryController` rausnehmen